org.brlcad.BRL_CAD.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: org.brlcad.BRL_CAD
+runtime: org.kde.Sdk
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: mged
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /share/doc
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: BRL-CAD
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      cflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
+        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized
+      cxxflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
+        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized
+    config-opts:
+      - -DBRLCAD_BUNDLED_LIBS=ON
+      - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DBRLCAD_EXTRADOCS:BOOL=OFF
+      - -DBRLCAD_ENABLE_OSG:BOOL=ON
+      - -DBUILD_TESTING:BOOL=OFF
+    sources:
+      - type: git
+        url: https://github.com/BRL-CAD/brlcad.git
+        commit: dc7c0b221cd74c13a3a4a6c74c679bf91cc8da42
+        tag: rel-7-34-2
+        x-checker-data:
+          type: git
+          tag-pattern: ^rel-([\d-]+)$
+
+    # The files disappear from source folders after install, so I cannot use post-install
+  - name: BRL-CAD-metainfo-desktopfile
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 mged.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.mged.desktop
```

Comment:
this should be just `${FLATPAK_ID}.desktop`. same for the icon. (based on what `command` is)

Reply:
ok, I will use mged.desktop as ${FLATPAK_ID}.desktop